html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form id = 'coding-form'>
+     <div>Email:</div>
+     <label for="email">
+      <input type="email" id="email" name="user_email">
+     </label>
```

Comment:
If you're going to put the `input` inside the `label`, then the `for` attribute isn't needed. It's either use the `for` attribute OR nest the `input` inside the `label`